include/fastdds/dds/rpc/RemoteExceptionCode_t.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef FASTDDS_DDS_RPC__REMOTEEXCEPTIONCODE_T_HPP
+#define FASTDDS_DDS_RPC__REMOTEEXCEPTIONCODE_T_HPP
+
+#include <cstdint>
+
+namespace eprosima {
+namespace fastdds {
+namespace dds {
+namespace rpc {
+
+/**
+ * Enumeration of possible error codes that can be returned by a remote service.
+ * Extracted from DDS-RPV v1.0 - 7.5.2 Mapping of Error Codes.
+ */
+enum class RemoteExceptionCode_t : uint32_t
```

Comment:
Currently `fastddsgen` is generating *enum class MyEnum : int32_t`. Should we change this?